packages/devtools/lib/src/timeline/event_details.dart

```diff
@@ -212,18 +229,7 @@ class _UiEventDetails extends CoreElement {
     final Spinner spinner = Spinner()..clazz('cpu-profile-spinner');
     add(spinner);
 
-    final Response response =
-        await serviceManager.service.getCpuProfileTimeline(
-      serviceManager.isolateManager.selectedIsolate.id,
-      event.startTime,
-      event.duration,
-    );
-
-    cpuProfileData = CpuProfileData(response);
-
-    if (showCpuFlameChart) {
-      (flameChart as CpuFlameChart).update(cpuProfileData);
-    }
+    await drawFlameChart();
```

Comment:
add a try catch and explain what happened if the data failed to load.

Reply:
done.